### PR TITLE
feat(sleepasandroid): remove `unlock-premium` patch (#1172)

### PR DESCRIPTION
**Pull request by Artaud**
Remove Sleep as Android patch as per conversation in https://github.com/revanced/revanced-patches/pull/1172

### CODE REVIEW

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -197,14 +197,6 @@ The official Patch bundle provided by ReVanced and the community.
 | `promo-code-unlock` | Disables the validation of promo code. Any code will work to unlock all features. | all |
 </details>
 
-### 📦 `com.urbandroid.sleep`
```
[the remark this answers is not in the document]
Comment (Artaud) [reply README.md:199]: Okay

[the remark this answers is not in the document]
Comment (oSumAtrIX) [reply README.md:199]: Basically only commit to the source files, the rest is autogenerated.

**Comment by oSumAtrIX** [comment]
Is this supposed to be closed?

**Review by oSumAtrIX — APPROVED** [review]

**Comment by Artaud** [comment]
Many thanks for the merge and especially for responsiveness.

**Comment by oSumAtrIX** [comment]
No problem, it is currently staged in the development branch to be merged into main. Thanks

**Comment by Canny1913** [comment]
> I was cleaning up the commits and it closed on its own at some point.

The PR will close automatically if there is no commits to merge.

**Pull request merged**